cmd/bootstrap/transit/main.go

```diff
@@ -45,6 +44,7 @@ var (
 	// default files to upload for all role type
 	filesToUpload = []string{
 		bootstrap.PathNodeInfoPub,
+		bootstrap.PathRootProtocolStateSnapshot, // push root protocol state snapshot
```

Comment:
If previously this logic uploaded the root block, seal, etc. files, and we are now writing the root-snapshot file in the same directory as we previously wrote those files as of https://github.com/onflow/flow-go/pull/424, then I don't think this is correct. The file tree would look like:
```
- $BOOTDIR/PathNodeInfoPub
|
+- root-snapshot.json
```

where previously it looked like:
```
- $BOOTDIR/PathNodeInfoPub
|
+- /root-block.json
+- /root-seal.json
+- /root-result.json
```

meaning, we still only need to upload the `PathNodeInfoPub` directory.